fele-client-service/cli/scripts/network.js

```diff
@@ -1,28 +1,61 @@
-const { createDatabase, deleteDatabase, insertToDatabase } = require('../../utils/db')
+const {
+  createDatabase,
+  deleteDatabase,
+  insertToDatabase,
+} = require("../../utils/db");
 const path = require("path");
-const fs = require('fs');
-const { createNetwork, deleteNetwork } = require('../../client-api/scripts/network')
+const fs = require("fs");
+const {
+  createNetwork,
+  deleteNetwork,
+} = require("../../client-api/scripts/network");
 
-const USER_WORKSPACE = "../../../tmpworkspaceforuser/"
+const USER_WORKSPACE = "../../../tmpworkspaceforuser/";
 
-async function createNetworkCLI(networkConfig, networkName) {
-    
-    if(networkConfig.includes(".json")) { //When networkConfig is a file
-        const fileName = USER_WORKSPACE+networkConfig;
-        networkConfig = require(fileName); //Get contents of file as object
-        networkConfig = JSON.stringify(networkConfig) //Convert object to string
-    } else {
-        networkConfig = networkConfig.replace(/\\/g,'')
+async function useNetworkCLI(username, localOrg, networkName) {
+  const { localUsers, felenetworks } = localOrg
+  let feleUser = {}
+  const localUserIndex = localUsers.findIndex((localUser) => (localUser.username === username));
```

Comment:
Can we separate this mapping logic from createNetworkCLI ? We can reuse this code for REST Api implementation as well. It will be clean too!

Reply:
Sure